Ensure the location is string when calling subprocesses. This is for the compatibility of Python 3.8-.
